### PR TITLE
Added RepeatedTest annotation to xtext.testing

### PR DESCRIPTION
**Pull request by szarnekow**
Can be used to run tests in a loop for n times

### CODE REVIEW

**Comment by kthoms** [comment]
This class makes use of JUnit 4 rules API. I know that there is already the Flaky rule in this package, but can we consider to move this new class to a "rules" package for clearer separation?

**Comment by kthoms** [comment]
Separation of JUnit 4 only API. We have the same with the "extensions" package for JUnit 5 only extensions.

**Comment by cdietrich** [comment]
id like it to keep as is

File: org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/RepeatedTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	/**
+	 * Test rule used along with {@link RepeatTest} to let a test loop for a couple of rounds.
+	 */
+	public static class Rule implements TestRule {
+
+		private final boolean quiet;
+
+		/**
+		 * @param quiet
+		 *            set to true if log output is desired
```
Comment (szarnekow) [inline org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/RepeatedTest.java]: Meh ... JavaDoc is bogus - will fix it immediatly

**Pull request merged**